src/main/java/com/cdancy/bitbucket/rest/domain/branch/Matcher.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.cdancy.bitbucket.rest.domain.branch;
+
+import com.google.auto.value.AutoValue;
+import org.jclouds.json.SerializedNames;
+
+@AutoValue
+public abstract class Matcher {
+
+    public enum MatcherId {
+        release("RELEASE", "Release", "MODEL_CATEGORY", "Branching model category"),
+        develop("development", "Development", "MODEL_BRANCH", "Branching model branch"),
+        master("production", "Production", "MODEL_BRANCH", "Branching model branch");
```

Comment:
Caps for enum names. Was it intentional to put `RELEASE` in call caps and then `development` not in all caps?